### PR TITLE
Add object detection label configs for cooking

### PR DESCRIPTION
**Pull request by hdefazio**

### CODE REVIEW

File: config/object_labels/recipe_tea.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
+version: "1"
+title: "Tea"
+# Note: ids 5-20 are shared with coffee
```
Comment (joshanderson-kw) [inline config/object_labels/recipe_tea.yaml:3]: Does this mean that ids 5-20 in this file and the coffee files should match? They look different

**Review by joshanderson-kw — COMMENTED** [review]
Where are these labels coming from? Is this just what we've been using for training?

Also, I see the notes about shared labels in several files, but the labels don't seem to match up.

File: config/object_labels/recipe_tea.yaml (hunk 1 above)
Comment (hdefazio) [reply config/object_labels/recipe_tea.yaml:3]: Just updated the ids across all the files

**Comment by hdefazio** [comment]
> Where are these labels coming from? Is this just what we've been using for training?

The coffee ones are the same ones that we've been using. The rest are new.

**Review by joshanderson-kw — APPROVED** [review]
LGTM!

**Pull request merged**